packages/cli/src/command-implementations/project-command.ts

```diff
@@ -152,19 +159,22 @@ export abstract class ProjectCommand<C extends ProjectCommandDefinition> extends
     let { default: definition } = utils.require.requireJsCode<{ default: sdk.IntegrationDefinition }>(artifact.text)
     definition = resolveInterfaces(definition)
     validateIntegrationDefinition(definition)
-    return definition
+    return { definition, bpLintDisabled }
   }
 
   private async _readInterfaceDefinitionFromFS(
     projectPaths: utils.path.PathStore<'workDir' | 'interfaceDefinition'>
-  ): Promise<sdk.InterfaceDeclaration | undefined> {
+  ): Promise<({ definition: sdk.InterfaceDeclaration } & LintIgnoredConfig) | undefined> {
     const abs = projectPaths.abs
     const rel = projectPaths.rel('workDir')
 
     if (!fs.existsSync(abs.interfaceDefinition)) {
       return
     }
 
+    const tsContent = await fs.promises.readFile(abs.interfaceDefinition, 'utf-8')
+    const bpLintDisabled = tsContent.startsWith('/* bplint-disable */')
```

Comment:
**nitpick**: this may cause an issue if the disable directive comes after the import statements, if there are blank lines or whitespace before, or if there are other directives before (eslint, typescript, prettier, ...). We could instead read line by line until we hit the directive or the project instantiation:

```typescript
import fs from 'fs'
import readline from 'readline'

namespace BpLintDirectives {
  export const isBpLintDisabled = async (filePath: string) => {
    const { readInterface, readStream } = _createReadInterface(filePath)
    
    try {
      return await _checkBpLintStatus(readInterface)
    } finally {
      readInterface.close()
      readStream.close()
    }
  }

  const _createReadInterface = (filePath: string) => {
    const readStream = fs.createReadStream(filePath, 'utf-8')
    const readInterface = readline.createInterface({ input: readStream })
    return { readInterface, readStream }
  }

  const _checkBpLintStatus = async (readInterface: readline.Interface) => {
    for await (const line of readInterface) {
      if (line.includes('bplint-disable')) return true
      if (line.includes('Definition({')) break
    }
    return false
  }
}
```

```typescript
const bpLintDisabled = BpLintDirectives.isBpLintDisabled(abs.interfaceDefinition)
```

Reply:
what about a simpler
```ts
/\/\* bplint-disabled \*\//.test(tsContent)
```
?

Reply:
I'll merge without and comeback to it if needed